types/jsonwebtoken/index.d.ts

```diff
@@ -77,6 +78,19 @@ export interface DecodeOptions {
     complete?: boolean;
     json?: boolean;
 }
+
+export interface SecretOrPublicKeyCallback {
```

Comment:
These should be `type`s not interfaces.

Reply:
Thanks @SimonSchick, I'll fix that.

Reply:
@SimonSchick Updated those callback interfaces to callback types.